sklearn/multiclass.py

```diff
@@ -203,6 +215,53 @@ def fit(self, X, y):
 
         return self
 
+    def partial_fit(self, X, y, classes=None):
+        """Partially fit underlying estimators
+        
+        Should be used when memory is inefficient to train all data.
+        Chunks of data can be passed in several iteration.
+        
+        Parameters
+        ----------
+        X : (sparse) array-like, shape = [n_samples, n_features]
+            Data.
+        y : (sparse) array-like, shape = [n_samples] or [n_samples, n_classes]
```

Comment:
To be consistent make this `(n_samples,)`, `(n_samples, n_classes)` here and everywhere.